tools/start-local-vm

```diff
@@ -116,8 +121,9 @@ parse_args() {
     [[ -n ${arch} ]] || usage_error 'Architecture needs to be set via either --arch or BUILDSYS_ARCH.'
     [[ -n ${variant} ]] || usage_error 'Variant needs to be set via either --variant or BUILDSYS_VARIANT.'
 
+    # shellcheck disable=SC2155  # This is safe even with a masked return code
```

Comment:
nit: hypothetically, if `uname` failed, the return code would be masked by the local assignment. IMO it's better practice to just always declare locals first before any assignment.